test/functional/VM_Test/src/j9vm/test/xlpcodecache/XlpCodeCacheOptionsTestRunner.java

```diff
@@ -140,7 +140,7 @@ protected void populateXlpOptionsList() {
 			xlpOptionsList.add(new XlpOption("-Xlp64K -Xlp:codecache:pagesize=16M", 16 * ONE_MB, XlpUtil.XLP_PAGE_TYPE_NOT_USED, true));
 			xlpOptionsList.add(new XlpOption("-Xlp:codecache:pagesize=16M -Xlp64K", 64 * ONE_KB, XlpUtil.XLP_PAGE_TYPE_NOT_USED, true));
 			xlpOptionsList.add(new XlpOption("-Xlp16M -Xlp:codecache:pagesize=7M", 7 * ONE_MB, XlpUtil.XLP_PAGE_TYPE_NOT_USED, true));
-			xlpOptionsList.add(new XlpOption("-Xlp:codecache:pagesize=7M -Xlp16M", 16 * ONE_KB, XlpUtil.XLP_PAGE_TYPE_NOT_USED, true));
+			xlpOptionsList.add(new XlpOption("-Xlp:codecache:pagesize=7M -Xlp16M", 16 * ONE_MB, XlpUtil.XLP_PAGE_TYPE_NOT_USED, true));
```

Comment:
Just to clarify, this change doesn't affect the failing test in #11104 right? It doesn't look related to the output in https://github.com/eclipse-openj9/openj9/issues/11104#issue-736400627. Did you catch it by reading the code or is there a test failure that pointed you to it? If you caught it by reading I'm wondering why the test itself didn't catch it.

Reply:
I think this change affects the failing test in #11104 because I caught this typo by inspecting a [test failure](https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/9007/), which included the [OMR fix](https://github.com/eclipse/omr/pull/5995) for the [default xlpOption](https://github.com/eclipse-openj9/openj9/blob/e495256d209c71f42c8472b267407fe5061c86bd/test/functional/VM_Test/src/j9vm/test/xlpcodecache/XlpCodeCacheOptionsTestRunner.java#L97) in `xlpOptionsList`. 

For each `xlpOption` in `xlpOptionsList`, the test seems to verify the value of `Xlp:codecache:pagesize=` in the output of `-verbose:sizes`.

The [test failure](https://hyc-runtimes-jenkins.swg-devops.com/view/OpenJ9%20-%20Personal/job/Pipeline-Build-Test-Personal/9007/) indicated that the [OMR fix](https://github.com/eclipse/omr/pull/5995) helped pass the first (default) `xlpOption` in `xlpOptionsList` and revealed a typo in another `xlpOption` that occurs later in `xlpOptionsList`.

Reply:
OK, that makes sense. It's good that the tests exposed both problems.